package.json

```diff
@@ -30,6 +30,7 @@
     "iflow-lodash": "^1.1.24",
     "iflow-react-router": "^1.2.1",
     "jest": "^20.0.4",
+    "jest-cli": "^20.0.4",
```

Comment:
Nope, I didn't anyway. I usually use `yarn test` (or npm) which means there's no need for the global install. Not that i'm against it, I just liked encapsulating all deps (even globally installed ones) in package.json.